portal-frontend/src/app/features/applications/edit-submission/proposal/pfrs-proposal/pfrs-proposal.component.ts

```diff
@@ -88,10 +88,10 @@ export class PfrsProposalComponent extends FilesStepComponent implements OnInit,
         this.submissionUuid = applicationSubmission.uuid;
 
         this.alreadyRemovedTableData = {
-          volume: applicationSubmission.soilAlreadyRemovedVolume ?? 0,
-          area: applicationSubmission.soilAlreadyRemovedArea ?? 0,
-          averageDepth: applicationSubmission.soilAlreadyRemovedAverageDepth ?? 0,
-          maximumDepth: applicationSubmission.soilAlreadyRemovedMaximumDepth ?? 0,
+          volume: applicationSubmission.soilAlreadyRemovedVolume ?? undefined,
```

Comment:
This will prevent the user from storing empty data if it was previously populated

Reply:
Just tested, and the form still saves the value if '0' was entered. Also, using `undefined` is the default behaviour of 'Soil to be Removed / Fill to be Placed' table. For some reason, that table was set to `undefined`, and 'Soil Already Removed / Fill Already Placed' table was '0' previously.

Reply:
Have you tried using "empty" (delete the value completely from the field), not 0?

Reply:
Tested together and there seems to be something in place to handle undefined value already 👍